packages/amplify_datastore/example/lib/codegen/Comment.dart

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
nit: I think we should eventually change the paths on these to match what codegen outputs.

Reply:
that is part of Kyle's PR, it will be getting merged here soon.